test/cctest/test_environment.cc

```diff
@@ -163,3 +173,11 @@ static void at_exit_callback_ordered2(void* arg) {
   EXPECT_FALSE(called_cb_ordered_1);
   called_cb_ordered_2 = true;
 }
+
+static void at_exit_js(void* arg) {
+  v8::Isolate* isolate = static_cast<v8::Isolate*>(arg);
+  v8::HandleScope handle_scope(isolate);
+  v8::Local<v8::Object> obj = v8::Object::New(isolate);
+  assert(!obj.IsEmpty());  // Assert VM is still alive.
+  assert(obj->IsObject());
+}
```

Comment:
Maybe `at_exit_cb_called++` so you can verify that the callback was actually invoked, like in the addon test.

Reply:
Sure, done